.github/workflows/vtk-pre-test.yml

```diff
@@ -5,7 +5,7 @@ on:
   push:
     branches:
       # To resolve issues on VTK master and test, use this branch name pattern
-      - "maint/vtk-master*"
+      - "maint/tox*"
```

Comment:
Changed temporarily to test that vtk master job is correctly running

Reply:
The vtk-master jobs run as expected, seeing the same errors as in another PR that runs the same job https://github.com/pyvista/pyvista/actions/runs/14483232798/job/40624180614?pr=7430

See https://github.com/pyvista/pyvista/actions/runs/14763559674